### DISCUSSION TITLE
Any restrictions on using MOOSE

### DISCUSSION THREAD

**Discussion by Oops-Qiao** [idaholab/moose#26567] [Q&A Getting Started]
Dear MOOSE colleagues,

Do we have any resctrictions on using MOOSE license regarding the users' background, such as countries from Iran etc.?
Anyone from the world can freely use MOOSE, right? Thank you.

Best regards,
John

**Comment by permcody** [comment]
Open source licenses do not discriminate against different people, groups, or organizations. As long as the end user works within the bounds of the applicable license, in MOOSE's case that's LGPLv2.1, then our software can be used around the world.
